pulsar-io/jdbc/sqlite/src/test/java/org/apache/pulsar/io/jdbc/SqliteJdbcSinkTest.java

```diff
@@ -63,6 +63,7 @@
  * Jdbc Sink test
  */
 @Slf4j
+@Test(groups = "flaky")
```

Comment:
I wonder if pulsar-io has proper handling for flaky tests. I think flaky group is currently handled only for pulsar-broker.

Reply:
It's better to omit this change for now.